build.cmd

```diff
@@ -1,108 +1,220 @@
+  call :dotnet_pack
+  exit /b %errorlevel%
+
+:build_microsoft_dotnet_xunit_performance_analysis_cli
+setlocal
+  cd %~dp0src\cli\Microsoft.DotNet.xunit.performance.analysis.cli
+  call :dotnet_pack
+  exit /b %errorlevel%
+
+:build_samples_classlibrary_net46
+setlocal
+  cd %~dp0samples\ClassLibrary.net46
+  call :dotnet_build
+  exit /b %errorlevel%
+
+:build_samples_simpleperftests
+setlocal
+  cd %~dp0samples\SimplePerfTests
+  call :dotnet_build
+  exit /b %errorlevel%
+
+:nuget_pack_src
+setlocal
+  cd %~dp0src
+  call %DotNet% nuget pack xunit.performance.nuspec                -p Configuration=%BuildConfiguration% --version=%PackageVersion% --output-directory %OutputDirectory% --symbols  || exit /b 1
+  call %DotNet% nuget pack xunit.performance.runner.Windows.nuspec -p Configuration=%BuildConfiguration% --version=%PackageVersion% --output-directory %OutputDirectory% --symbols  || exit /b 1
+  exit /b 0
+
+:build_xunit_performance_api
+setlocal
+  cd %~dp0src\xunit.performance.api
+  call :dotnet_pack
+  exit /b %errorlevel%
+
+:build_tests_simpleharness
+setlocal
+  cd %~dp0tests\simpleharness
+  call :dotnet_build  || exit /b 1
+  net.exe session 1>nul 2>&1 || (
+    call :print_error_message Cannot run simpleharness test because this is not an administrator window
+    exit /b 1
+  )
+  call %DotNet% run -c %BuildConfiguration% bin\%BuildConfiguration%\netcoreapp1.0\simpleharness.dll                                                                                                        || exit /b 1
+  exit /b %errorlevel%
+
+:dotnet_build
+  echo/
+  echo/  ==========
+  echo/   Building %cd%
+  echo/  ==========
+  call :remove_directory bin                                                    || exit /b 1
+  call :remove_directory obj                                                    || exit /b 1
+  call %DotNet% restore                                                         || exit /b 1
+  call %DotNet% build -c %BuildConfiguration% --version-suffix %VersionSuffix%  || exit /b 1
+  exit /b 0
+
+:dotnet_pack
+  echo/
+  echo/  ==========
+  echo/   Packing %cd%
+  echo/  ==========
+  call :remove_directory bin                                                                                                || exit /b 1
+  call :remove_directory obj                                                                                                || exit /b 1
+  call %DotNet% restore                                                                                                     || exit /b 1
+  call %DotNet% build -c %BuildConfiguration% --version-suffix %VersionSuffix%                                              || exit /b 1
+  call %DotNet% pack  -c %BuildConfiguration% --version-suffix %VersionSuffix% --output %OutputDirectory% --include-symbols || exit /b 1
+
+  :: FIXME: pack sources does not work with the current mixed version of the Tracing library (EXCEPTION THROWN).
```

Comment:
Not that I know of.

---
In reply to: [100385566](https://github.com/Microsoft/xunit-performance/pull/183#discussion_r100385566) [](ancestors = 100385566)